Automattic/jetpack
# Instant Search: add search form to main content area

#### Changes proposed in this Pull Request:

In order to offer a search form and filtering on mobile, we need to render the markup for those page elements in the main search content area (above search results).

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
No - it's part of the Instant Search prototype and merges into `instant-search-master`.

#### Testing instructions:






Follow the setup instructions in https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md.

Using the smallest breakpoint, verify that a search box appears and is usable.

Make sure it remains present when there are no results.

<img width="322" alt="Screen Shot 2019-12-03 at 11 32 36" src="https://user-images.githubusercontent.com/17325/70001313-d125ac80-15c1-11ea-84ec-463ffd3d9f62.png">
<img width="327" alt="Screen Shot 2019-12-03 at 11 31 28" src="https://user-images.githubusercontent.com/17325/70001314-d125ac80-15c1-11ea-95e7-d9cde23a366f.png">


#### Proposed changelog entry for your changes:
Not required.
